src/components/layout/Finishable/__specs__/FinishableContainer.spec.js

```diff
@@ -50,9 +52,34 @@ describe('components | FinishableContainer', () => {
         const props = mapStateToProps(state, ownProps)
 
         // then
-        expect(props).toStrictEqual({
-          offerCanBeBooked: false,
-        })
+        expect(props.offerCanBeBooked).toBe(true)
+      })
+
+      it('should return false when offer is not bookable anymore and current user has not booked it', () => {
+        // given
+        getIsBooked.mockReturnValue(false)
+        const state = {
+          data: {
+            bookings: [{ id: 'A1', stockId: 'B1' }],
+            offers: [{ id: 'C1', isBookable: false }],
+            stocks: [{ id: 'B1', offerId: 'C1' }],
+          },
+        }
+        const ownProps = {
+          match: {
+            params: {
+              bookingId: 'A1',
+            },
+          },
+        }
+
+        const expectedResult = false
+
+        // when
+        const props = mapStateToProps(state, ownProps)
+
+        // then
+        expect(props.offerCanBeBooked).toBe(expectedResult)
```

Comment:
```suggestion
        expect(props.offerCanBeBooked).toBe(false)
```
on a pris l'habitude de mettre directement la valeur dans le expected (au moins sur la webapp)

Reply:
👍